otterdog/eclipse-tractusx.jsonnet

```diff
@@ -1547,6 +1547,46 @@ orgs.newOrg('automotive.tractusx', 'eclipse-tractusx') {
         },
       ],
     },
+    orgs.newRepo('tractusx-identityhub') {
+      allow_merge_commit: true,
+      allow_update_branch: false,
+      delete_branch_on_merge: false,
+      allow_auto_merge: true,
+      gh_pages_build_type: "legacy",
+      gh_pages_source_branch: "gh-pages",
+      gh_pages_source_path: "/",
+      has_discussions: true,
+      private_vulnerability_reporting_enabled: true,
+      web_commit_signoff_required: false,
+      environments: [
+        orgs.newEnvironment('github-pages') {
+          branch_policies+: [
+            "gh-pages"
+          ],
+          deployment_branch_policy: "selected",
+        },
+      ],
+    },
+    orgs.newRepo('tractusx-issuerservice') {
```

Comment:
why is the `tractusx` prefix needed? is there already an implementation in some other GH org (like in the case of the identityHub)?
```suggestion
    orgs.newRepo('ssi-issuer-service') {
```

There are quite some ssi related components in Tractus-X now, to all find them in [one search](https://github.com/orgs/eclipse-tractusx/repositories?type=all&q=ssi), adding the `ssi-` would be beneficial.

Reply:
see above.

Reply:
IMO, using "ssi-" instead of (or in addition to) "tractusx-" makes the purpose less apparent:

- Tractus-X indicates it is part of the TX project
- The repos are distributions of EDC IdentityHub/IssuerService and follow the same convention as Tractus-X EDC
- The repos provide runtimes that implement the DCP protocol. DCP purposely does not use the term SSI because it is too vague and its design center is "decentralized claims" not "self-sovereign identities". 

As an option, we could name the repos "tractusx-edc-*", which would be consistent with "tractusx-edc" .

Reply:
I don't quite agree for instance the fact that it is located in the eclipse-tractusx GH indicates that it is part of the TX project at the same time I do not feel it would be productive to continue this discussion further, just leave it as is and provide an explanation to this comment https://github.com/eclipse-tractusx/.eclipsefdn/pull/116#issuecomment-2615244792